modules/support/client/components/BlockMemberLink.component.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * A link to beport member
+ * @param {string} username - the username of the user to report
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import '@/config/client/i18n';
+import { useTranslation } from 'react-i18next';
+
+import * as api from '../api/block.api';
+
+
+
+
+export default function BlockMemberLink({ username }) {
+  const { t } = useTranslation('support');
+
+  async function handleBlockMember({ username }) {
+    const data = await api.block(username); 
+  }
+
+  return (
+    <a
+      href={`/support?report=${username}`}
```

Comment:
The `href` isn't needed here, so we can probably switch the whole `<a>` tag to a `<button>` instead?

To get it to render like it's a link you can add `btn btn-link` into `className`. Not sure if it does the trick but let's see how it looks :-)